docs/types.rst

```diff
@@ -864,18 +864,8 @@ or create a new memory array and copy every element.
 Array Literals
 ^^^^^^^^^^^^^^
 
-Array literals (also called inline arrays) are arrays that are written as an
-expression and are not assigned to a variable right away.
-
-::
-
-  pragma solidity >=0.4.16 <0.6.0;
-
-  contract C {
-    function f() public pure {
-      uint[] memory a;
-    }     
-  }
+An array literal is a list of zero or more array elements, enclosed in square
```

Comment:
I don't think zero elements are possible. At least the compiler would not be able to assign a type to it.

Reply:
Apart from that, it sounds really good :)